### DISCUSSION TITLE
Is voice call supported with EM series? VoLTE

### DISCUSSION THREAD

**Discussion by agentpatience** [danielewood/sierra-wireless-modems#14] [General] [answered]
Hi Daniele, Thanks for all the great information about these modems!

My provider offers VoLTE... I have EM7355 which doesn't seem to support voice but I am willing to buy one that does. Do you know which modems do? Have you tried making any voice calls with these?

**Comment by danielewood** [comment, marked as answer]
Calling over these modems isn't a thing, yet. Maybe in another five years it will be on some new model 5G modem. 

You can do SMS, but voice is strictly going to be a VoIP solution.
